### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by kiryldz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 ## v0.10.0
 
-- Add support for Snapdragon 855
+- Added support for Snapdragon 855
+- Improved `VisionView` lifecycle management
+- Updated API of `VisionManager`
```
Comment (yunikkk) [inline CHANGELOG.md]: Let's also add notes about fixed leaks, `Detections` rename, cones detection and new segmentation classes

Comment (RingerJK) [inline CHANGELOG.md]: shouldn't we add `New segmentation model recognizes...` as in [`ios`](https://github.com/mapbox/vision-ios-examples/blob/master/CHANGELOG.md) ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,18 @@
 
 ## v0.10.0
 
-- Add support for Snapdragon 855
+- Added support for Snapdragon 855
+- `VisionView` renders now with OpenGL ES
+- Changed public API of `VisionView`
+- Changed `VisionView` lifecycle if `VisionManager` is set
+- Change internal camera lifecycle
```
Comment (RingerJK) [inline CHANGELOG.md]:
```suggestion
- Changed internal camera lifecycle
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,18 @@
 
 ## v0.10.0
 
-- Add support for Snapdragon 855
+- Added support for Snapdragon 855
+- `VisionView` renders now with OpenGL ES
```
Comment (RingerJK) [inline CHANGELOG.md]:
```suggestion
- Changed `VisionView` renders now with OpenGL ES
```

**Pull request merged**